eng/pipelines/libraries/fuzzing/deploy-to-onefuzz.yml

```diff
@@ -35,12 +35,13 @@ extends:
 
         - powershell: |
             cd $(Build.SourcesDirectory)
-            ./build.cmd clr+libs+packs+host -rc Checked -c Debug
+            ./build.cmd clr+libs -rc Checked -c Debug
           displayName: Build runtime (checked + debug)
 
         - powershell: |
             cd $(fuzzerProject)
-            $(dotnetPath) publish -o publish
+            $(dotnetPath) build-server shutdown
```

Comment:
Was this row added intentionally?

Reply:
Yes, the bot ran into issues without it.
```
[00:27:58] D:\runner-dir\runtime\.dotnet\sdk\9.0.100-rc.1.24452.12\Microsoft.Common.CurrentVersion.targets(4858,5): warning MSB3026: Could not copy "D:\runner-dir\runtime\artifacts\obj\ILLink.Tasks\Debug\net9.0\ILLink.Tasks.dll" to "D:\runner-dir\runtime\artifacts\bin\ILLink.Tasks\Debug\net9.0\ILLink.Tasks.dll". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\runner-dir\runtime\artifacts\bin\ILLink.Tasks\Debug\net9.0\ILLink.Tasks.dll' because it is being used by another process. The file is locked by: ".NET Host (4880)" [D:\runner-dir\runtime\src\tools\illink\src\ILLink.Tasks\ILLink.Tasks.csproj::TargetFramework=net9.0]
```

Since the logic here is pretty much the same, I expect the pipeline would hit it as well.

Reply:
OK, thank you!